.github/ISSUE_TEMPLATE/3-documentation.yml

```diff
@@ -0,0 +1,44 @@
+name: '📚 - Documentation'
+title: '📚 - '
+description: Report an issue in Maskito's documentation
+labels: ['documentation']
+
+body:
+  - type: input
+    id: affected-url
+    attributes:
+      label: What is the affected URL?
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      placeholder: |
+        How do you trigger this bug?
+        Please walk us through it step by step.
+        Please provide a screenshot if possible.
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: browser-specific
+    attributes:
+      label: Which browsers have you used?
+      description: You may select more than one.
+      options:
+        - label: Chrome
+        - label: Firefox
+        - label: Safari
+        - label: Edge
+
+  - type: checkboxes
+    id: operating-systems
```

Comment:
nit: I'm not sure we need information about operating system in this type of bug-report 🤔

Reply:
`iOS/Android`

Reply:
> `iOS/Android`

@splincode do you think that it is better to keep this section ?

Reply:
I think so, but as you wish

Reply:
I keep it (to be similar to Taiga UI)